HW_4/task_3.py

```diff
@@ -0,0 +1,41 @@
+#"""
+#Задать натуральное число k.
+#Сформируйте многочлен (полином) степени k со случайными коэффициентами из промежутка от 0 до 100, включительно.
+#Многочлен вывести в консоль и записать в файл.
+
+#Ввод: значение типа <int>
+#Вывод: значение типа <str>, файл с одной строкой.
+
+#Пример:
+#2
+#2x^2 + 4x + 5 = 0
+#"""
+
+from random import randint
+import random
+
+k = int(input('Введите число k: '))
+
+result = ''
+temp = []
+for i in range(k):
+    temp.append(randint(0, 100))
+
+
+znak = ['+', '-']
+i = 0
+j = 0
+while k > 1:
+    if temp[i] != 0:
+        result += (f'{temp[i]}x**{k}{random.choice(znak)}')
```

Comment:
А еще по заданию вывод должен быть со знаком `^`
```suggestion
        result += (f'{temp[i]}x^{k}{random.choice(znak)}')
```

Reply:
Пожалуйста, читайте внимательно условия задачи. Нужно было еще, помимо записи в файл, вывести в консоль. А вот записывать список коэффициентов вообще не нужно было